<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="cityInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Grenoble</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>City name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="radiusInput">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Export radius</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="highwaysGroup">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Highways</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="motorwayCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Motorway</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="trunkCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Trunk</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="primaryCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Primary</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="secondaryCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Secondary</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="tertiaryCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Tertiary</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Waters</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="riverCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>River</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="lakeCheckBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>87</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lake</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="generateMapButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="text">
     <string>Generate map</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="svgFrame">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>371</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Style</string>
    </property>
    <widget class="QComboBox" name="wStyleComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
